packages/x-date-pickers/src/AdapterDateFnsJalali/AdapterDateFnsJalali.ts

```diff
@@ -138,7 +138,7 @@ const defaultFormats: AdapterFormats = {
   monthShort: 'MMM',
   dayOfMonth: 'd',
   weekday: 'EEEE',
-  weekdayShort: 'EEE',
+  weekdayShort: 'EEEEE',
```

Comment:
shouldn't this also be `EEEEEE` 🤔 
```suggestion
  weekdayShort: 'EEEEEE',
```

Reply:
You were probably checking the first commit changeset. 🙈